src/Digdir.Domain.Dialogporten.WebApi/Program.cs

```diff
@@ -54,7 +55,7 @@ static void BuildAndRun(string[] args)
     builder.Host.UseSerilog((context, services, configuration) => configuration
         .MinimumLevel.Warning()
         .MinimumLevel.Override("Microsoft.EntityFrameworkCore", Serilog.Events.LogEventLevel.Fatal)
-        .MinimumLevel.Override("ZiggyCreatures.Caching.Fusion", builder.Environment.EnvironmentName == "test" ?
+        .MinimumLevel.Override("ZiggyCreatures.Caching.Fusion", builder.Environment.IsDevelopment() ?
             Serilog.Events.LogEventLevel.Debug : Serilog.Events.LogEventLevel.Warning)
```

Comment:
```suggestion
```

IMAO mener jeg vi burde opte in for debug level logging i de tilfeller vi forsøke å finne et problem. Det være seg lokalt, eller et eller annet eksternt miljø.

Reply:
Enig i det. Men jeg bare kommenterer ut koden i stedet for å slette den, så er det litt lettere å kjapt slå på ved behov. På sikt bør vi se på om det finnes mekanismer for å kunne endre på minimumlevel runtime (evt med en app restart) ved behov.